release/airflow/cloud_build.template.json

```diff
@@ -7,8 +7,8 @@
           "path": "/output"
 	}
       ],
-      "name": "gcr.io/istio-testing/istio-builder:0.5.7",
```

Comment:
Why are these CB files moved to airflow dir? They are not specific to airflow right?
Can they be in release/cloudbuild instead?

Reply:
they are used by airflow side of code to execution on cloud build
look for run_build in airflow_scripts, the json files are passed as parameters there